casbah/src/main/scala/akka/contrib/persistence/mongodb/CasbahPersistenceJournaller.scala

```diff
@@ -34,11 +35,17 @@ class CasbahPersistenceJournaller(driver: CasbahMongoDriver) extends MongoPersis
     val batch = writes.map(write => Try(driver.serializeJournal(Atom[DBObject](write, driver.useLegacySerialization))))
     if (batch.forall(_.isSuccess)) {
       val bulk = journal.initializeOrderedBulkOperation
```

Comment:
For both, let's return the results of the insert into the journal, not the insert into the realtime collection.